apps/cnspec/cmd/scan.go

```diff
@@ -374,8 +376,33 @@ This example connects to Microsoft 365 using the PKCS #12 formatted certificate:
 		}
 
 		logger.DebugDumpJSON("report", report)
+		// TODO: ONLY FOR TESTING PURPOSES REMOVE ME
+		if fileName := os.Getenv("DEBUG_SAVE_REPORT_COLLECTION_FILE"); fileName != "" {
```

Comment:
this should automattically happen when `log.Debug().Enabled()` is enabled. See logging helper function we have here https://github.com/mondoohq/cnquery/blob/main/logger/debug.go

Reply:
Yes, I just wanted a quick way to get the binary protobuf for testing w/o having to convert the json. I'll remove this entirely